### DISCUSSION TITLE
MindTheGapps和OpenGApp有何区别？

### DISCUSSION THREAD

**Discussion by Sunrongguo2008** [LSPosed/MagiskOnWSALocal#179] [Q&A] [answered]
构建WSA时，如何在MindTheGapps和OpenGApp之间选择？

**Comment by Howard20181** [comment, marked as answer]
上下方向键切换 空格键选中

**Reply by Sunrongguo2008** [reply]
MindTheGapps版本和OpenGApp版本有何区别？

**Reply by Howard20181** [reply]
对我来说没有区别，有些人想要谷歌全家桶就要装OpenGApps
我只需要核心服务

**Comment by MC-dusk** [comment]
[lineage](https://wiki.lineageos.org/gapps/)两种都提供了，新版本都是MindTheGapps，只有安卓7和6才给的OpenGApps，感觉是mind要简单小一些，适合刷ROM的时候一起刷了，简单给一个GMS环境，如果不需要谷歌全家桶的话。不过OpenGApps也有pico版就是了。

---

https://github.com/opengapps/opengapps/issues/759
https://github.com/opengapps/opengapps/wiki/Package-Comparison
https://gitlab.com/MindTheGapps/vendor_gapps
https://github.com/MindTheGapps
